js/services.ts

```diff
@@ -42,6 +42,10 @@ module jsonforms.services {
         resolveModelPath(instance:any, path:string): any
     }
 
+    export interface ISchemaGenerator {
```

Comment:
Would it make sense to unify the ISchemaGenerator and the IUISchemaGenerator such that having a single IGenerator interface? If more 'default' generators are added in the future it might get cumbersome if we need inject each one of them. What do you think?